firebase/README.md

```diff
@@ -2,37 +2,52 @@
 
 We use Firebase for various features:
 
-* Alert Signups use Cloud Firestore for storing subscription info as well as
+- Alert Signups use Cloud Firestore for storing subscription info as well as
   progress info while sending alerts.
-* We use Firebase Hosting with Firebase Functions for generating share / export
+- We use Firebase Hosting with Firebase Functions for generating share / export
   images dynamically on-demand.
 
 ## Installing Firebase CLI
+
 Run `yarn global add firebase-tools` or `npm install -g firebase-tools` to
 install the Firebase CLI globally.
 
 ## Deploying
+
 To deploy everything (Firestore rules / indexes, Hosting, Functions) to the
 dev environment (covidactnow-dev), run:
 
-```
+```sg
 firebase -P dev deploy
 ```
 
 To deploy to the staging environment (covidactnow-staging), run:
-```
+
+```sh
 firebase -P staging deploy
 ```
 
 To deploy to the prod environment (covidactnow-prod), run:
-```
+
+```sh
 firebase -P prod deploy
 ```
 
 ## Testing Hosting / Functions
-To test hosting / functions locally:
 
+Before running the functions locally for the first time, you will need to login and choose the environment to run
+
+```sh
+# This will open the browser to authorize and authenticate the Firebase CLI
+firebase login
+
+# This will use the development configuration by default
+firebase use dev
```

Comment:
I'm a bit surprised you needed to do this.  I thought it was already the default based on our .firebaserc file I don't really know how that works, so 🤷

Reply:
the CLI detected the config but showed me a message telling me that I had to choose one to use by default 🤷